dluhc-component-library/src/components/maps/components/DatasetLayer.tsx

```diff
@@ -18,6 +20,7 @@ const DatasetLayer = ({ area, dataset, zIndex = 1 }: DatasetlayerProps) => {
   if (isLoading || isError) {
     return null;
   }
+  const features = new GeoJSON().readFeatures(data);
```

Comment:
Do we know how computationally expensive this is? Is it worth using `useMemo`? @samornsby

Reply:
In the `GeoJSON().readfeatures()` code it looks like it loops over each element to convert it into a feature, but that is just assigning fields at that index to an object. I could see `useMemo` being worth it if a list is involved, but I'm not 100% sure.